testsuite/tools/expect_test.ml

```diff
@@ -76,8 +76,8 @@ let match_expect_extension (ext : Parsetree.extension) =
         let normal, principal =
           match e.pexp_desc with
           | Pexp_tuple
-              [ a
-              ; { pexp_desc = Pexp_construct
```

Comment:
Here also we should probably preserve the old behavior by checking for `None`.

Well, I say the old behavior - I guess the old one ignored labels because they were in attributes, but I think it's fair to say the intent was to match on only unlabeled tuples?  Or let me know if you disagree.